repositories/settings/core.go

```diff
@@ -22,3 +37,46 @@ func (c *core) setResolution(width, height int32) {
 func (c *core) getResolution() (int32, int32) {
 	return c.width, c.height
 }
+
+func (c *core) setFromReader(reader io.Reader) error {
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		line := scanner.Text()
+		err := scanner.Err()
+		if err != nil {
+			return err
+		}
+		if strings.Trim(line, " \t") == "" {
```

Comment:
https://pkg.go.dev/strings@go1.17#TrimSpace

Reply:
This has been updated.